### DISCUSSION TITLE
deploying workspaces into separate subscriptions?

### DISCUSSION THREAD

**Discussion by TonyWildish-BH** [microsoft/AzureTRE#3815] [Ideas]
One feature we think we might like is the ability to deploy workspaces into different subscriptions, each attached to their own billing methods. This can be useful for groups that have their own billing accounts set up already, and don't want the hassle of having us charge them back for their resource usage. Has anyone considered this option?

**Comment by marrobi** [comment]
Tracked here - https://github.com/microsoft/AzureTRE/issues/1073

@SvenAelterman was looking into this, not sure if it's progressed.

**Comment by TonyWildish-BH** [comment]
thanks, I'd missed that one when I looked for anything related. I'll close this discussion now.

**Comment by SvenAelterman** [comment]
Tony, I will keep the issue Marcus referenced updated as updates are available.
